src/ui/components/Timeline.qml

```diff
@@ -524,9 +530,10 @@ Item {
         width: parent.width; 
         anchors.bottom: parent.bottom;
         ToolTip {
-            text: qsTr("%1 to zoom horizontally, %2 to zoom vertically, double click to reset zoom")
-                    .arg(Qt.platform.os == "osx"? "<b>" + qsTr("Command+Scroll") + "</b>" : "<b>" + qsTr("Ctrl+Scroll") + "</b>")
-                    .arg(Qt.platform.os == "osx"? "<b>" + qsTr("Control+Scroll") + "</b>" : "<b>" + qsTr("Alt+Scroll") + "</b>");
+            text: qsTr("%1 to zoom horizontally, %2 to zoom vertically, %3 to pan, double click to reset zoom")
+                    .arg(Qt.platform.os == "osx"? "<b>" + qsTr("Scroll") + "</b>" : "<b>" + qsTr("Scroll") + "</b>")
+                    .arg(Qt.platform.os == "osx"? "<b>" + qsTr("Control+Scroll") + "</b>" : "<b>" + qsTr("Alt+Scroll") + "</b>")
+                    .arg(Qt.platform.os == "osx"? "<b>" + qsTr("Command+Scroll") + "</b>" : "<b>" + qsTr("Ctrl+Scroll") + "</b>");

```

Comment:
let's extract `"<b>" + qsTr() + "</b>"` outside of the if so it becomes
`.arg("<b>" + qsTr(Qt.platform.os == "osx"? "Command+Scroll" : "Ctrl+Scroll") + "</b>");`